config/initializers/foreman.rb

```diff
@@ -19,6 +19,7 @@
   end
 
   Setting.descendants.each(&:load_defaults)
+  Setting.validate_all
```

Comment:
You answered your concern basically, I don't want to iterate through all of them. Seems this is broken by design. Will try different approach here.